src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchain.cs

```diff
@@ -88,11 +88,9 @@ private Task InitBlockchain()
                 : NullWitnessCollector.Instance;
 
             setApi.MainStateDbWithCache = getApi.DbProvider.StateDb
-                .Innermost // TODO: PRUNING what a hack here just to pass the actual DB
                 .Cached(Trie.MemoryAllowance.TrieNodeCacheCount)
                 .WitnessedBy(witnessCollector);
             IKeyValueStore? codeDb = getApi.DbProvider.CodeDb
-                .Innermost // TODO: PRUNING what a hack here just to pass the actual DB
```

Comment:
will it work with like RPC db or beam sync?

Reply:
worth to beam sync goerli once on master, I am running validator and fasts ynced xdai on this ine